src/markdown-pages/docs/writing-own-tests/running-test-modes.md

```diff
@@ -0,0 +1,106 @@
+
+### <a name="whenremote"></a>When to run a test remotely
+
+Running a test remotely is useful in the following scenarios:
+
+
+- <b>Tests need to run in bulk, in parallel, often submitted by different automation jobs or people, in an environment where resources are limited and need to be managed between tests. For example, ports, slots of processing capability, files, and sessions.</b> 
+
+If you are able to dynamically provision a system, then running tests from a local JVM can work well, as the system under test might have few resource contraints and can be used exclusively by the tests before being de-provisioned. However, if you want to run many test in parallet, you cannot do so reliably from a single system using the local JVM method on one user account. Any contention to name test runs uniquely, or other stateful changes in the test framework might cause the tests to clash, over-write each other, fail, or produce unexpected results. Running multiple tests in series can avoid some of these issues but running large numbers of tests in this way can take a long time. 
+
+To reliably run many tests in parallel, deploy your tests to the Galasa Ecosystem, letting the ecosystem manage the test runs, and the Galasa Manager components manage any shared or constrained resources. 
+
+- <b>Test results and reports need gathering from a single point. For example, when test results need reporting or exporting to another report-generating system, or when bug investigation can proceed by independent inspection test results and artifacts.</b>
+
+A problem faced in large-scale use of the local JVM method of test invocation is how to gather test results from disparate machines or environments, so that the test results can be combined and form test reports for system health monitoring. This is something that running tests within a Galasa ecosystem handles well, as the results of each test are stored centrally. Test results can be shared with others, enabling independent inspection and debugging of run artifacts. 
+
+- <b>When the clean-up of resources in the system under test is an important requirement</b>
+
+When you run tests in a Galasa Ecosystem, resources are cleaned up by the Managers that are called during abnormal test exit conditions. When a locally run test stops prematurely, the Manager code that handles the clean-up of allocated resources might not run, so resources are not cleaned up. Over time, this can result in performance degradation of the system under test. 
+
+
+### <a name="whenlocal"></a>When to run a test locally
+
+Running a test locally is useful when you are doing the following types of task:
+
+- Developing tests, and running those tests against a non-production environment.
+
+- Running ad-hoc tests against a development or non-production target system.
+
+- Running many tests in series against a target system which can withstand the load that is placed on it.
```

Comment:
Seems that we need a 'when to run tests within an ecosystem' section to contrast with this 'when to run tests locally' ?